src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/OrderStateController.php

```diff
@@ -398,4 +415,15 @@ private function getErrorMessages(\Exception $e)
             ),
         ];
     }
+
+    protected function addMultishopWarningFlashMessage(): void
+    {
+        if ($this->get('prestashop.adapter.multistore_feature')->isActive()) {
+            $this->addFlash(
+                'info',
+                $this->trans('Note that this feature is available in all shops context only. It will be added to all your stores.',
+                    'Admin.Notifications.Info')
+            );
```

Comment:
```suggestion
            $this->addFlash(
                'info',
                $this->trans(
                    'Note that this feature is available in all shops context only. It will be added to all your stores.',
                    'Admin.Notifications.Info'
                )
            );
```